mapgen.lua

```diff
@@ -1,10 +1,23 @@
+-- check for MineClone2
+local mcl = minetest.get_modpath("mcl_core")
+
+if mcl then
+    item1 = "mcl_core:water_source"
```

Comment:
Names like `item_water_source` or `item_dirt` would improve readability a lot for future changes.